### DISCUSSION TITLE
Reset to default

### DISCUSSION THREAD

**Discussion by Denethor85** [EdgeTX/edgetx#1514] [Q&A]
in the new version 2.6.0 there is no eeprom reset button, so how can I reset each setting to default?  I tried reinstalling the firmware, wiping the SD card, but none of this seems to work ....

Thanks

**Comment by pfeerick** [comment]
With 2.6.0, wiping the SD card will reset to defaults as the settings are stored on the SD card now. There would be an edge case with B&W radios though, as they would import all the settings EEPROM to SD on the first boot ... I suppose we would have to make it so there was some way of detecting that you don't want to import after deleting the config files.

**Comment by Denethor85** [comment]
Yes, this would be optimal. In fact, I have a Taranis X7 and I found all the settings even if i deleted the sd card
________________________________
